observations/obs_converters/radar/README.rst

```diff
@@ -45,19 +45,119 @@ script:
+as listed below. The pyOPAWS tool requires compilation and expects
+GNU fortran (gcc) to be loaded.
+
+::
+ 
+  1) ncarenv/24.12 (S)      5) libfabric/1.15.2.0
+  2) craype/2.7.31          6) cray-mpich/8.1.29
+  3) gcc/12.4.0             7) hdf5/1.12.3
+  4) ncarcompilers/1.0.0    8) netcdf/4.9.2
+
+1.  Load the environment management system for python (conda) as: 
+
+::
+
+   module load conda
+
+2. Navigate to your local pyOPAWS directory {pyOPAWS_DIR}, then activate the
+   pyOPAWS python environment as:
+
+::
+
+   cd {pyOPAWS_DIR}
+   conda env create -f pythnon4opaws.yml
+   conda activate opaws_env
+
+3. Confirm list of ``opaws_env`` packages:
+
+::
+
+  conda list --name opaws_env
+
+
+  # packages in environment at conda-envs/opaws_env:
+  #
+  # Name                    Version                   Build  Channel
+  _libgcc_mutex             0.1                 conda_forge    conda-forge
+  _openmp_mutex             4.5                       2_gnu    conda-forge
+  aiobotocore               2.21.1             pyhd8ed1ab_0    conda-forge
+  aiohappyeyeballs          2.6.1              pyhd8ed1ab_0    conda-forge
+  aiohttp                   3.11.13         py310h89163eb_0    conda-forge
+  aioitertools              0.12.0             pyhd8ed1ab_1    conda-forge
+  aiosignal                 1.3.2              pyhd8ed1ab_0    conda-forge
+  alabaster                 1.0.0                    pypi_0    pypi
+  ..
+  .. and many more
+
+4. To run the converter script (``opaws2D.py``) first compile the fortran cressman routine
+   which assumes the GNU compiler was installed:
+
+::
+
+  python fcompile_cressman.py
+
+Confirm ``cressman.cpython-310-x86_64-linux-gnu.so`` has been built in {pyOPAWS_DIR}.
+
+
+
+5. (OPTIONAL) Follow the `example <https://github.com/louiswicker/pyOPAWS?tab=readme-ov-file#usage-examples>`__
+   instructions to generate a diagnostic plot of radar data as:
+
+::
+
+   python lvl2_plot.py -f KOUN20110524_224700_V06 -q None -p 1 -u phase
+
+View the file {pyOPAWS_DIR}/images/KOUN_20110524_224700_0.68DEG.png which contains
+a multi-panel image of reflectivity, unfolded radial velocity, spectrum width,
+Z-dr, RHO_H-V, PHI-DP. 
+ 
+
+6. Follow the `example <https://github.com/louiswicker/pyOPAWS?tab=readme-ov-file#usage-examples>`__
+   instructions to convert the NEXRAD file (KOUN20110524_224700_V06) to DART `obs_seq` format. 
+   Note that the -w flag indicates the DART `obs_seq*.out` files will be generated.
+
+::
+  
+  python opaws2d.py -f KOUN20110524_224700_V06 -q None -p 1 -w -u phase
+
+Confirm the following files within the {pyOPAWS_DIR}/opaws_files/ were created:
+
+::
+
+  KOUN_20110524_224700_01.pdf
+  obs_seq_KOUN_20110524_224700_VR.out
+  obs_seq_KOUN_20110524_224700_RF.out
+  obs_seq_KOUN_20110524_224700.nc
+
+Note that two DART `obs_seq*.out` files have been generated that include
+the radial velocity and reflectivity observations respectively. 
```

Comment:
Be careful here of writing documentation for non-dart software. 

Loading modules on Derecho is covered in the ncar hpc documentation ( & the list of modules here will be out-of-date in the next few months), the installation and running of opaws is in the OPAWS documentation. 

If the OPAWS documentation is not sufficient for users to get started, I'd suggest putting a pull request https://github.com/louiswicker/pyOPAWS.  

```suggestion
```

Reply:
> Be careful here of writing documentation for non-dart software.
> 
> Loading modules on Derecho is covered in the ncar hpc documentation ( & the list of modules here will be out-of-date in the next few months), the installation and running of opaws is in the OPAWS documentation.
> 
> If the OPAWS documentation is not sufficient for users to get started, I'd suggest putting a pull request https://github.com/louiswicker/pyOPAWS.

Other than the modules becoming out of date -- the rest of the instructions should be fairly stable, and are more comprehensive than OPAWS.    I prefer to keep them, but don't feel super strongly about it.